### ISSUE TITLE
Segmentation fault in WSL2

### ISSUE THREAD

**Issue by hrueger** [Julusian/node-midi#3]
Hi @Julusian,
it's me again. I've just ran into a very strange issue: On my Windows 10 PC with Ubuntu in WSL2, I get a `Segmentation Fault` every time I try to list the Inputs.

It's very easy to reproduce:
```JavaScript
const easymidi = require("easymidi")
try {
    console.log(easymidi.getInputs())
} catch {}
```

I'd like this to just return nothing or fail, so that I can catch the error. However, I get the following:
```
hannes@Hannes-Desktop:~/tests$ node
Welcome to Node.js v16.18.1.
Type ".help" for more information.
> const easymidi = require("easymidi")
undefined
> try { easymidi.getInputs() } catch {  }
ALSA lib seq_hw.c:466:(snd_seq_hw_open) open /dev/snd/seq failed: No such file or directory

MidiInAlsa::initialize: error creating ALSA sequencer client object.

undefined
> Segmentation fault
```

`Segmentation Fault` happens about 1s after the `try ... catch` call.

It's fine for me that MIDI is not working inside WSL (that's probably an issue of my setup, I haven't really looked into this yet, but it shouldn't crash so hard.

This happens with Ubuntu on WSL2 on Windows 10:
```
hannes@Hannes-Desktop:~/tests$ uname -a
Linux Hannes-Desktop 5.10.102.1-microsoft-standard-WSL2 #1 SMP Wed Mar 2 00:30:59 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
hannes@Hannes-Desktop:~/tests$ cat /etc/os-release
PRETTY_NAME="Ubuntu 22.04.1 LTS"
NAME="Ubuntu"
VERSION_ID="22.04"
VERSION="22.04.1 LTS (Jammy Jellyfish)"
VERSION_CODENAME=jammy
ID=ubuntu
ID_LIKE=debian
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
UBUNTU_CODENAME=jammy
```

Interestingly, on my Windows 11 machine, it works just fine (meaning `getInputs()` throws an error but does not crash with `Segmentation Fault`.

Do you have any idea?

**Comment by Julusian** [comment]
This should be fixed now in v3.0.0

**Comment by hrueger** [comment]
Can confirm that the bug is resolved now, thanks a ton!
